config/set/contentrepository-90.php

```diff
@@ -435,6 +436,23 @@
      */
     $rectorConfig->rule(WorkspaceGetNameRector::class);
 
+    /**
+     * Neos\ContentRepository\Domain\Service\NodeTypeManager
+     */
+    $rectorConfig->rule(NodeTypeManagerAccessRector::class);
+    // createNodeType(nodeTypeName: string): NodeType
+    $methodCallToWarningComments[] = new MethodCallToWarningComment(\Neos\ContentRepository\Domain\Service\NodeTypeManager::class, 'createNodeType', '!! NodeTypeManager::createNodeType() was never implemented and is removed in Neos 9.0.');
+    // getNodeType(nodeTypeName: string): NodeType
+    // --> Compatible with 9.0
+    // getNodeTypes([includeAbstractNodeTypes: bool = true]): NodeType[]
+    // --> Compatible with 9.0
+    // getSubNodeTypes(superTypeName: string, [includeAbstractNodeTypes: bool = true]): NodeType[]
+    // --> Compatible with 9.0
+    // hasNodeType(nodeTypeName: string): bool
+    // --> Compatible with 9.0
+    // overrideNodeTypes(completeNodeTypeConfiguration: array): void
+    // --> Compatible with 9.0
```

Comment:
```suggestion
    // overrideNodeTypes(completeNodeTypeConfiguration: array): void
    // --> Compatible with 9.0
```

this is not api and was also removed as far as i know

Reply:
but very fresh
https://github.com/neos/neos-development-collection/commit/24ffeff863eb202b3c7494641dbd73ddd74dfcb1

Reply:
I've added a comment now. See [fc2a988](https://github.com/neos/rector/pull/121/commits/fc2a98863e8fdd8933d4d31632a30c29e5497df0)